frontend/components/footer/footer.css

```diff
@@ -0,0 +1,9 @@
+.footer__copyrights__bg {
```

Comment:
move the footer.css file to the styles folder in the root directory and import it in the index.css file